__tests__/linkdrop.spec.ts

```diff
@@ -58,11 +58,12 @@ describe(`Running on ${Runner.getNetworkFromEnv()}`, () => {
           gas: tGas("50"),
         }
       );
-      // @ts-ignore
+      const bob = runtime.getAccount(new_account_id, false);
```

Comment:
Note for later cleanup: I don't like `false` as a bald second argument; I think we should have made this take an object as the second argument.

```js
getAccount(id, { addSubaccountPrefix: false })
```

Much more readable that way (I don't have to go look at the function definition to figure out what's going on)

Reply:
Good point. Moving this into account also has the benefit of knowing which subaccount it could automatically check if subaccount.